bdist_rpm.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+VERSION=`grep "Version:.*[0-9]" slurm.spec | tr -s " " |  awk '{print $2;}'`
+RELEASE=`grep "%global rel.*[-1-9]" slurm.spec | tr -s " " | awk '{print $3}'`
+
+if [ ${RELEASE} -gt 1 ]; then
+    SUFFIX=${VERSION}-${RELEASE}
+else
+    SUFFIX=${VERSION}
+fi
+
+git archive --format=tar.gz -o /tmp/slurm-${SUFFIX}.tar.gz --prefix=slurm-${SUFFIX}/ 17.11.ug
+
+mv /tmp/slurm-${SUFFIX}.tar.gz ${HOME}/rpmbuild/SOURCES
+
+cp slurm.spec ${HOME}/rpmbuild/SPECS
+cd ${HOME}/rpmbuild/SPECS
+
+rpmbuild -ba slurm.spec --with=mysql --with=lua --with=hwloc --with=numa
```

Comment:
i don't like hardocded release versions from spec files:
can you add a
```
gittag=`git log --format=%ct.%h -1`
rpmbuild --define "gittag $gittag"
```
and set in the spec file `Release: %{rel}%{gittag}%{?dist}.ug'

untested, but should work fine

Reply:
Done.